packages/components/demo/scss/_theme.scss

```diff
@@ -10,22 +10,13 @@
     $theme-map--with-custom-properties: map-merge(
       $theme-map--with-custom-properties,
       (
-        $key: var(--#{$key}, $value),
+        $key: var(--cds-#{$key}, $value),
```

Comment:
Note: this part is temporary and should be replaced by @emyarod's feature flag work here 👍